layer/json/VkLayer_gfxreconstruct.json.in

```diff
@@ -303,33 +305,33 @@
                             "key": "log_level",
                             "env": "GFXRECON_LOG_LEVEL",
                             "label": "Level",
-                            "description": "Specify the highest level message to log. Options are: debug, info, warning, error, and fatal. The specified level and all levels listed after it will be enabled for logging. For example, choosing the warning level will also enable the error and fatal levels.",
+                            "description": "Specify the minimum severity level of messages to log. Options are: debug, info, warning, error, and fatal. The specified level and all higher levels will be enabled for logging. For example, choosing the warning level will also enable the error and fatal levels.",
                             "type": "ENUM",
                             "flags": [
                                 {
                                     "key": "debug",
-                                    "label": "debug, info, warning, error, fatal",
-                                    "description": ""
+                                    "label": "debug",
```

Comment:
I find these new labels more confusing. 

The Vulkan developer can't really understand the order of each value: "debug, info, warning, error, fatal". With the previous naming, the Vulkan developer only considers what kind of message is outputred.

I think in the future this setting should be refactored as a FLAGS, and the user would be able to select indecently each area of messages just like the other layers.

Reply:
I agree, its still a bit awkward. I can revert this section until we come up with a better solution